src/notebook/plugin.ts

```diff
@@ -179,6 +194,42 @@ class NotebookFileHandler extends AbstractFileHandler {
     return ['.ipynb']
   }
 
+  executeSelectedCell(): void {
+    for (let w of this.widgets) {
+      if (w.hasClass('jp-mod-focus')) {
```

Comment:
It is defined in `jupyter-js-docmanager`.  We'd be duplicating the `node.contains` logic there.